### PR TITLE
3 Crear endpoint para recibir JSON

### LINKED ISSUE #3: Create endpoint for receiving Json

### PR DESCRIPTION
**Pull request by anntGFT**

### CODE REVIEW

**Review by AxseGFT — APPROVED** [review]

File: src/main/java/com/db/transform/controller/TransformController.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+public class TransformController {
+
+    @Autowired
+    TransformService service;
+
+    @GetMapping(value = "/trades")
```
Comment (gnugomez) [inline src/main/java/com/db/transform/controller/TransformController.java]: you should use ```/trades/saveFile``` or something like this as we said this morning

File: src/main/java/com/db/transform/service/TransformService.java [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+import org.springframework.web.reactive.function.client.WebClient;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+@Service
+public class TransformService {
+
+    WebClient webClient;
```
Comment (gnugomez) [inline src/main/java/com/db/transform/service/TransformService.java]: This code will need to be refactored later to separate webClient to a repository and replace it with the correct things, so i think we can exclude this from this pr for now.

File: src/main/resources/tradeName-cobdate.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<ArrayList>
```
Comment (gnugomez) [inline src/main/resources/tradeName-cobdate.xml]: While you use this file for testing purposes you should store this file in ```src/test/resources/...```

**Review by gnugomez — CHANGES_REQUESTED** [review]
Please check those things before merging

**Review by Jubaal — APPROVED** [review]

**Review by gnugomez — APPROVED** [review]

**Pull request merged**